modules/exploits/multi/misc/legend_bot_exec.rb

```diff
@@ -30,6 +30,7 @@ def initialize(info = {})
       'License'        => MSF_LICENSE,
       'References'     =>
         [
+          [ 'AKA', 'Shellshock' ],
```

Comment:
This is an exploit for the bot, so there's no Shellshock here. The bot was likely deployed through other vulns, too.